go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/cosmos/gogoproto v1.4.11
 	github.com/cosmos/ibc-go/modules/capability v1.0.0
 	github.com/cosmos/ibc-go/v8 v8.0.0
+	github.com/cosmos/interchain-security/v3 v3.2.0-consumer-rc0.0.20231123140529-1819e73f6197
```

Comment:
Given that it isn't yet released, nor tagged, I don't think we can merge this.
Or we need to figure out our tagging and release branch story quick.
Otherwise this will get included in the next patch release if we merge.